content/doc/book/pipeline/syntax.adoc

```diff
@@ -257,33 +257,31 @@ For example, if you want a pod with a Kaniko container inside it, you would defi
 ----
 agent {
     kubernetes {
-        label podlabel
-        yaml """
+        defaultContainer 'kaniko'
+        yaml '''
 kind: Pod
-metadata:
-  name: jenkins-agent
```

Comment:
You definitely do not want to specify the pod name. Not sure if this is quietly overridden, or if it actively breaks something.